fineract-provider/src/main/java/org/apache/fineract/infrastructure/creditbureau/service/ThitsaWorksCreditBureauIntegrationWritePlatformServiceImpl.java

```diff
@@ -434,10 +480,14 @@ public String getCreditBureauConfiguration(Integer creditBureauId, String config
 
             creditBureauConfigurationValue = configurationParameterValue.getValue();
             if (creditBureauConfigurationValue.isEmpty()) {
-                baseDataValidator.reset().failWithCode("creditBureau.configuration." + configurationParameterName + ".is.not.available");
-                throw new PlatformApiDataValidationException(
-                        "creditBureau.Configuration." + configurationParameterName + ".is.not.available",
-                        "creditBureau.Configuration.is.not.available", dataValidationErrors);
+                /*
+                 * baseDataValidator.reset().failWithCode("creditBureau.configuration." + configurationParameterName +
+                 * ".is.not.available"); throw new PlatformApiDataValidationException( "creditBureau.Configuration." +
+                 * configurationParameterName + ".is.not.available", "creditBureau.Configuration.is.not.available",
+                 * dataValidationErrors);
+                 */
+                throw new PlatformDataIntegrityException("creditBureau.Configuration." + configurationParameterName + ".is.not.available",
+                        "creditBureau.Configuration." + configurationParameterName + ".is.not.available");
```

Comment:
@rrpawar96 did you mean to change this, or this a left over from a local test that you did not mean to change? If it should change, perhaps you would prefer to just remove instead of comment out the bean validation which you don't seem to want to do here anymore? (But why?)

Reply:
Yes, it was the change in the implementation of the type of Exception, (PlatformDataIntegrityException instead of PlatformApiDataValidationException.) PlatformDataIntegrityException was more suitable to throw this exception.